apis/fluentbit/v1alpha2/plugins/output/open_telemetry_types.go

```diff
@@ -0,0 +1,86 @@
+package output
+
+import (
+	"fmt"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// OpenTelemetry is An output plugin to submit Metrics to an OpenTelemetry endpoint,
+// allows taking metrics from Fluent Bit and submit them to an OpenTelemetry HTTP endpoint.
+type OpenTelemetry struct {
+	// IP address or hostname of the target HTTP Server, default `127.0.0.1`
+	Host string `json:"host,omitempty"`
+	// TCP port of the target OpenSearch instance, default `80`
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=65535
+	Port *int32 `json:"port,omitempty"`
+	// Optional username credential for access
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Password for user defined in HTTP_User
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// Specify an HTTP Proxy. The expected format of this value is http://HOST:PORT. Note that HTTPS is not currently supported.
+	// It is recommended not to set this and to configure the HTTP proxy environment variables instead as they support both HTTP and HTTPS.
+	Proxy string `json:"proxy,omitempty"`
+	// Specify an optional HTTP URI for the target web server, e.g: /something
+	Uri string `json:"uri,omitempty"`
+	// Add a HTTP header key/value pair. Multiple headers can be set.
+	Header map[string]string `json:"header,omitempty"`
+	// Log the response payload within the Fluent Bit log.
+	LogResponsePayload *bool `json:"logResponsePayload,omitempty"`
+	// This allows you to add custom labels to all metrics exposed through the OpenTelemetry exporter. You may have multiple of these fields.
+	AddLabel     map[string]string `json:"addLabel,omitempty"`
+	*plugins.TLS `json:"tls,omitempty"`
+}
+
+// Name implement Section() method
+func (_ *OpenTelemetry) Name() string {
+	return "opentelemetry"
+}
+
+// Params implement Section() method
+func (o *OpenTelemetry) Params(sl plugins.SecretLoader) (*params.KVs, error) {
+	kvs := params.NewKVs()
+	if o.Host != "" {
+		kvs.Insert("Host", o.Host)
+	}
+	if o.Port != nil {
+		kvs.Insert("Port", fmt.Sprint(*o.Port))
+	}
+	if o.HTTPUser != nil {
+		u, err := sl.LoadSecret(*o.HTTPUser)
+		if err != nil {
+			return nil, err
+		}
+		kvs.Insert("HTTP_User", u)
+	}
+	if o.HTTPPasswd != nil {
+		pwd, err := sl.LoadSecret(*o.HTTPPasswd)
+		if err != nil {
+			return nil, err
+		}
+		kvs.Insert("HTTP_Passwd", pwd)
+	}
+	if o.Proxy != "" {
+		kvs.Insert("proxy", o.Proxy)
+	}
+	if o.Uri != "" {
+		kvs.Insert("uri", o.Uri)
```

Comment:
For the treatment of parameter initials, I suggest that they be uniformly capitalized.

Reply:
But those fields all is lower case in the [user guide docs](https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry).

Reply:
https://github.com/fluent/fluent-operator/pull/345/files/b2c0e2e1cd3bdffa7acfe3ebc1696eccc1188c16#diff-9ebeb7a15971d4a78c371b1113aa3aca09bcae1b4d2b1dd78775681ad9df02e6R46
But capitalization is used here, and I think it is better to be uniform.

Reply:
Done.